packages/nimble-blazor/build/generate-hybrid/source/NimbleBlazor.HybridWorkaround.js

```diff
@@ -1,3 +1,3 @@
-import { afterStarted } from '../../../NimbleBlazor/wwwroot/NimbleBlazor.lib.module';
+import { afterStarted } from '@ni/nimble-blazor/wwwroot/NimbleBlazor.lib.module';
```

Comment:
Can you undo this change. It creates an unnecessary coupling to the blazor build running first.

Reply:
Requires turning of a lint rule about relative imports from other packages, but that's fine.